sklearn/linear_model/_base.py

```diff
@@ -419,6 +420,13 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    solver : str, default="lsqr"
```

Comment:
`lsqr` is used for consistency with the Ridge module.

Reply:
No, those are different. `Ridge` has solver option `lsqr` which then calls `scipy.sparse.linalg.lsqr`. Here, we want to say `lsqrt` (or some other name), because it calls `scipy.linalg.lstsq`.